packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -58,9 +58,9 @@ const ALLOW_VIRTUAL_ELEMENTS = false;
 /**
  * Handle global tab presses so that we can patch tabindexes on the fly.
  */
-function _onKeyDownCapture(ev: KeyboardEvent) {
+function _onKeyDownCapture(this: Window, ev: KeyboardEvent) {
```

Comment:
```suggestion
function _onKeyDownCapture(window: Window, ev: KeyboardEvent) {
```
```suggestion
function _onKeyDownCapture(currentWindow: Window, ev: KeyboardEvent) {
```

??

Reply:
Actually if we rename that we would have to pass it explicit when add it as listener

Reply:
not sure i followed that.